mmdet3d/datasets/transforms/formating.py

```diff
@@ -147,15 +147,15 @@ def pack_single_results(self, results: dict) -> dict:
         if 'img' in results:
             if isinstance(results['img'], list):
                 # process multiple imgs in single frame
-                imgs = [img.transpose(2, 0, 1) for img in results['img']]
-                imgs = np.ascontiguousarray(np.stack(imgs, axis=0))
+                imgs = [to_tensor(img) for img in results['img']]
```

Comment:
Should also add comments to indicate why, and we should also check whether the array is contiguous?

Reply:
I will add comments. I have tested torch.permute always faster than np.transpose. So I think there is no need for check if tensor is contiguous

Reply:
test script
```
import time

import numpy as np
import torch

a = np.random.rand(1336, 3, 800)
b=a.transpose(0,2,1) # not contiguous
# b = np.random.rand(1366, 800, 3) # contiguous

np_time = 0
torch_time = 0
if_time =0 
for i in range(2000):
    if i < 1000:  # warm_up
        c = torch.from_numpy(np.ascontiguousarray(b.transpose(2, 0, 1)))
        d = torch.from_numpy(b).permute(2, 0, 1).contiguous()
    else:
        start_time = time.perf_counter()
        c = torch.from_numpy(np.ascontiguousarray(b.transpose(2, 0, 1)))
        np_time += time.perf_counter() - start_time

        start_time = time.perf_counter()
        d = torch.from_numpy(b).permute(2, 0, 1).contiguous()
        torch_time += time.perf_counter() - start_time

        start_time = time.perf_counter()
        if b.flags.c_contiguous:
            d = torch.from_numpy(b).permute(2, 0, 1).contiguous()
        else:
            c = torch.from_numpy(np.ascontiguousarray(b.transpose(2, 0, 1)))
        if_time += time.perf_counter() - start_time
print(f'torch time: {torch_time:.3f} numpy time:{np_time:.3f} if time:{if_time:.3f}')
```